viv-wallet-app/src/views/Payment.vue

```diff
@@ -15,12 +15,7 @@
                 <form class="payment-form">
                     <div class="element-block inline-bloc w33">
                         <label id="lbl-date" for="payment-date">DATE</label>
-                        <vc-date-picker
-                            mode="single"
-                            id="payment-date"
-                            v-model="date"
-                            class="values"
-                        />
+                        <vc-date-picker mode="single" id="payment-date" v-model="date" class="values" />
```

Comment:
oh! just realize we have a date picker already.
how is it better than the standard stuff we use on the consultant edit page?

Reply:
for info: in that page I set the date to "today" by default, it would be nice to do that either everywhere or nowhere for consistency.
tell me what you think about that ;)

Reply:
It was a formatting issue (I did not change this part).
Wael added a date picker. I'm not sure if we need it. The default date picker is pretty good (except on safari).
I agree with you. We have to be careful about consistency. Perhaps set the date to today everywhere.